service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOController.java

```diff
@@ -120,18 +119,16 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(@RequestHeader("autho
     @PostMapping("/about-to-submit")
     public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody CallbackRequest callbackRequest) {
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
-        final Map<String, Object> data = caseDetails.getData();
-        CaseData caseData = mapper.convertValue(data, CaseData.class);
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        CaseManagementOrder populatedCMO = draftCMOService.prepareCMO(
-            caseData, caseData.getCaseManagementOrder());
+        CaseManagementOrder populatedCMO = draftCMOService.prepareCMO(caseData, caseData.getCaseManagementOrder());
 
-        draftCMOService.removeTransientObjectsFromCaseData(data);
+        draftCMOService.removeTransientObjectsFromCaseData(caseDetails.getData());
 
-        data.put("caseManagementOrder", populatedCMO);
+        caseDetails.getData().put("caseManagementOrder", populatedCMO);
```

Comment:
would we update this to use `CASE_MANAGEMENT_ORDER_LOCAL_AUTHORITY.getKey()` once [PR-699](https://github.com/hmcts/fpl-ccd-configuration/pull/699/files) is merged?

Reply:
yep, that is absolutely the plan @Gogetter